.github/workflows/test.yml

```diff
@@ -30,9 +30,4 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
 
-      - name: Setup gotestsum
-        uses: autero1/action-gotestsum@v2.0.0
-        with:
-          gotestsum_version: 1.11.0
-
-      - run: gotestsum -f testname -- ./... -race
+      - run: go run gotest.tools/gotestsum@v1.11.0 -f testname -- ./... -race
```

Comment:
- name: Install gotestsume
  run: go install gotest.tools/gotestsum@latest

- name: Run gotestsum
  run: gotestsum -f testname -- ./... -race

Reply:
Can do, but does that have an advantage over go run?

Reply:
Doesn't need to compile if the release has binaries. It's also similar to what we do in other workflows.

Reply:
Go install always compiles, it won't download.  But in any case, I made the change.